lib/kitchen/directions/bake_chapter_glossary/v1.rb

```diff
@@ -21,9 +21,17 @@ def <=>(other)
       attr_reader :sortable
     end
 
-    def bake(chapter:, metadata_source:, append_to: nil, uuid_prefix: '')
+    def bake(chapter:, metadata_source:, append_to: nil, uuid_prefix: '', has_para: false)
       @glossary = []
 
+      # Use for books created by Adaptarr
```

Comment:
Looks good - just one question to make sure I understand. 

This is fixing a transforms issue in some content created by Adaptarr and the team thought it would be best to resolve it in the recipe? 

Maybe it would be good to also link to the issue in this comment so in case someone needs further explanation as to where the paragraph is coming from they can refer to the issue? (if I'm understanding this correctly lol)

Reply:
Sure :wink: I modified a comment.

In Adaptarr user have an ability to write description of the term in a few paragraphs (a few meanings). This is something that is not needed in Openstax books, but might be useful if we decide to use Adaptarr for another projects. So when you creating meaning of the term it puts your text automatically to paragraph. And this is something that we don't want to change in a tool.